java/util/src/main/java/com/google/protobuf/util/JsonFormat.java

```diff
@@ -408,6 +409,7 @@ public static Parser parser() {
     private final com.google.protobuf.TypeRegistry registry;
     private final TypeRegistry oldRegistry;
     private final boolean ignoringUnknownFields;
+    private final boolean lenient;
```

Comment:
You've made this a boolean, but this is actually just two of an infinite possible number of formats it might want to parse.